addons/website/views/website_templates.xml

```diff
@@ -1435,6 +1435,13 @@
     </xpath>
 </template>
 
+<!-- Area for blocks shared by all pages -->
+<template id="shared_blocks" inherit_id="website.layout" name="Shared blocks" active="True">
+    <xpath expr="//header" position="after">
+        <div id="o_shared_blocks" class="oe_empty"/>
```

Comment:
`oe_empty`, if I am not mistaken, is a class that only makes sense when used with `oe_structure` and which allow to indicate a big "add building block here" in edit mode when the area is empty. I don't think you need it here?